scalableminds/webknossos
# Public tracings

### Mailable description of changes:
 - tracings can be marked public to make them available without login

### Steps to test:
- disable devAutoLogin in config
- create explorative tracing
- make sure dataset is also publicly available
- share tracing and activate public sharing without login
- open sharing URL in a new incognito window without cookies / session
- tracing should still load

### Issues:
- fixes #1942

------
